nextflow_schema.json

```diff
@@ -8,86 +8,293 @@
+                "input"
+            ],
             "properties": {
                 "input": {
                     "type": "string",
                     "fa_icon": "fas fa-dna",
-                    "description": "Input FastQ files.",
-                    "help_text": "A glob pattern for input FastQ files. Should include at least one asterisk (*). For paired-end data, should contain curly brackets with two patterns differentiating the paired reads e.g. `*_R{1,2}.fastq.gz`"
-                },
-                "single_end": {
-                    "type": "boolean",
-                    "description": "Specifies that the input is single-end reads.",
-                    "fa_icon": "fas fa-align-center",
-                    "default": false,
-                    "help_text": "By default, the pipeline expects paired-end data. If you have single-end data, specify this parameter on the command line when you launch the pipeline. It is not possible to run a mixture of single-end and paired-end files in one run."
+                    "description": "Input DIA mzML files",
+                    "help_text": "Use this to specify the location of your input DIA mzML files. For example:\n\n```bash\n--input ''path/to/data/sample_*.mzML'\n```\n\nPlease note the following requirements:\n\n1. The path must be enclosed in quotes\n2. The path must have at least one `*` wildcard character\n\nIf left unspecified, a default pattern is used:  'data/sample_*.mzML'",
+                    "default": "data/sample_*.mzML"
                 },
                 "outdir": {
                     "type": "string",
                     "description": "The output directory where the results will be saved.",
-                    "default": "./results",
+                    "default": ".&#x2F;results",
                     "fa_icon": "fas fa-folder-open"
                 },
                 "email": {
                     "type": "string",
                     "description": "Email address for completion summary.",
                     "fa_icon": "fas fa-envelope",
-                    "help_text": "An email address to send a summary email to when the pipeline is completed.",
+                    "help_text": "Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits. If set in your user config file (`~/.nextflow/config`) then you don't need to specify this on the command line for every run.",
                     "pattern": "^([a-zA-Z0-9_\\-\\.]+)@([a-zA-Z0-9_\\-\\.]+)\\.([a-zA-Z]{2,5})$"
                 }
-            },
-            "required": [
-                "input"
-            ],
-            "fa_icon": "fas fa-terminal",
-            "description": "Define where the pipeline should find input data and save output data."
+            }
         },
-        "reference_genome_options": {
-            "title": "Reference genome options",
+        "spectral_library_generation": {
```

Comment:
just out of interest: where are these defaults from? maybe document?

Reply:
hmm.. do you mean specific ones or all parameter default values? I think I took them either from the OpenSwathWorkflow defaults or some of them are optimized parameters for orbitrap data that George Rosenberger has determined.